free.js

```diff
@@ -34,13 +34,22 @@ Free.prototype.chain = function(f) {
 
 const liftF = command => Free.Impure(command, Free.Pure)
 
+const IGNORE_VALUE = typeof Symbol != "undefined"
+  ? Symbol.for("free/ignore_value")
+  : "@@free/ignore_value"
+
```

Comment:
I dont know if that will have some bad effects as it's internal thing. but your question made me to think on much better solution which I have just pushed.
And if we move Monad's do function into Free.do we would not need to export this too.